docs/proposals/2017-11-17_-_Ratings.md

```diff
@@ -0,0 +1,163 @@
+# Problem
+
+A common genre of document across many publishers is a consumer review of products or media, such as movies, books, or gadgets. A consumer review is like any other article, but it often includes a formal rating of the item being reviewed. Rating values may be used:
+* to create renderable widgets on the site (Four gold star graphics and one gray star graphic for "4/5 stars") or
+* to sort content ("sort documents in this section by review score")
+
+There are further complications in a multi-tenant environment. Different publishers -- or even different desks or writers within a publisher -- may position the rating in different places within an article. Not all publishers or desks may use the same review system ("4/5 starts" vs "9.1 (out of 10)" vs "Recommended" or "Not Recommended")
+
+In some cases, like a round-up or a gift guide, a single article may also have multiple ratings for different items.
+
+This proposal attempts to address most of these cases by providing new content elements for these purposes.
+
+# Proposal
+
+Two new content elements that represent numeric and non-numeric ratings values.
+
+## New 'Rating' element
+
+A new content element `rating` will be added to indicate a numeric rating, with the following fields:
+
+* `rating` -- a float indicating the item's value (required)
+* `min` -- a float the minimum possible value of rating
+* `max` -- a float indicating the maximum possible value of rating
+* `units` -- a string describing the rating units (e.g., "stars," "thumbs")
+
+### Example
+```
+{
+  "type": "rating",
+  "rating": 4.5,
+  "min": 0,
+  "max": 5.0,
+  "units": "stars"
+}
+```
+
+## New 'Endorsement' element
+
+A new content element `endorsement` will be added to indicate text-only appraisals
+
+* `endorsement`: a string indicating the value (e.g. "Buy" or "Don't Buy") (required)
```

Comment:
Instead of having a unsortable endorsement field, why don't we add a displayName so that the translation of score text to Text value (like from the thumbs up and thumbs down example) doesn't have to happen at the rendering layer.

Reply:
I'm fine adding displayName, but I do think it's worth demarcating the difference between quantifiable / unquantifiable ratings

Reply:
I can't think of a rating system where the rating is unquantifiable. Can you provide an example?

Reply:
WashPost Pinnochio system

Reply:
It's up to the org to decide their ratings. So for books, you could have ratings of must read, don't read, maybe read. Or for movies go see or don't see.